crates/ruff_linter/src/rules/flake8_bandit/rules/hardcoded_sql_expression.rs

```diff
@@ -88,7 +87,6 @@ pub(crate) fn hardcoded_sql_expression(checker: &mut Checker, expr: &Expr) {
             };
             string.value.to_str().escape_default().to_string()
         }
-        // f"select * from table where val = {val}"
```

Comment:
Let's keep this comment as it shows an example of what we're trying to match here.